source/src_lcao/ELEC_nscf.h

```diff
@@ -23,7 +23,7 @@ class ELEC_nscf
 	ELEC_nscf();
 	~ELEC_nscf();
 
-	private:
+	//private:
```

Comment:
Please move the target function or variable to public, rather than delete private directly.

Reply:
If some code outside module_esolver should be changed, you should add annotation to describe why these code should be changed.